### PR TITLE
PageUrl generation #589

### PR DESCRIPTION
**Pull request by anatol-sialitski**

### CODE REVIEW

File: src/main/java/com/enonic/app/guillotine/graphql/helper/ParamsUrHelper.java [hunk 1]
```diff
@@ -0,0 +1,40 @@
+package com.enonic.app.guillotine.graphql.helper;
+
+import java.util.Map;
+import java.util.Objects;
+
+import com.google.common.collect.Multimap;
+
+public class ParamsUrHelper
```
Comment (rymsha) [inline src/main/java/com/enonic/app/guillotine/graphql/helper/ParamsUrHelper.java:8]: private constructor

File: src/main/java/com/enonic/app/guillotine/graphql/helper/ParamsUrHelper.java [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+{
+    public static void resolveParams( final Multimap<String, String> holder, final Object originalParams )
+    {
+        if ( originalParams instanceof Map )
```
Comment (rymsha) [inline src/main/java/com/enonic/app/guillotine/graphql/helper/ParamsUrHelper.java:12]: what if not? Why we silently skip other types?

**Review by rymsha — APPROVED** [review]

File: src/main/java/com/enonic/app/guillotine/graphql/helper/ParamsUrHelper.java (hunk 2 above)
Comment (anatol-sialitski) [reply src/main/java/com/enonic/app/guillotine/graphql/helper/ParamsUrHelper.java:12]: Yes, the same behavior in XP.
https://github.com/enonic/xp/blob/master/modules/lib/lib-portal/src/main/java/com/enonic/xp/lib/portal/url/AbstractUrlHandler.java#L48
https://github.com/enonic/xp/blob/master/modules/lib/lib-portal/src/main/java/com/enonic/xp/lib/portal/url/AbstractUrlHandler.java#L61

File: src/main/java/com/enonic/app/guillotine/graphql/helper/ParamsUrHelper.java (hunk 1 above)
Comment (anatol-sialitski) [reply src/main/java/com/enonic/app/guillotine/graphql/helper/ParamsUrHelper.java:8]: done

**Review by rymsha — APPROVED** [review]

**Pull request merged**